features/activity/cancel_try_cancel/feature.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from datetime import timedelta
+from typing import Optional
+
+from temporalio import activity, workflow
+from temporalio.client import Client, WorkflowHandle
+from temporalio.common import RetryPolicy
+from temporalio.exceptions import ActivityError, ApplicationError, CancelledError
+
+from harness.python.feature import Runner, register_feature
+
+
+@workflow.defn
+class Workflow:
+    def __init__(self) -> None:
+        self._activity_result: Optional[str] = None
+
+    @workflow.run
+    async def run(self) -> None:
+        # Start workflow
+        handle = workflow.start_activity(
+            cancellable_activity,
+            schedule_to_close_timeout=timedelta(minutes=1),
+            heartbeat_timeout=timedelta(seconds=5),
+            # Disable retry
+            retry_policy=RetryPolicy(maximum_attempts=1),
+            cancellation_type=workflow.ActivityCancellationType.TRY_CANCEL,
```

Comment:
I know we want to use this code for documentation but I think there's value in sharing this kind of configuration between the different implementations

Reply:
I saw that you have a RetryDisabled constant in Go.
Maybe make the entire activity config for long-running-at-most-once a constant in all harnesses?

Reply:
As we start to reuse, sure. Let me get to the next set and if that schedule to close and heartbeat and such remain the same, I'll definitely create something shared.